### PR TITLE
Add the udev rules to the installation prerequisites.

### PR DESCRIPTION
**Pull request by Peter-Easton**

### CODE REVIEW

**Comment by thestinger** [comment]
The `adbusers` group isn't required to permit using devices from local sessions, so you shouldn't need to include it. It's only needed to permit using devices that are plugged in locally from a remote session. That's generally how udev permissions work. It already grants permission to access all the local devices for local sessions.

**Comment by Peter-Easton** [comment]
Got it. I'll erase that part...

**Comment by thestinger** [comment]
@Peter-Easton It looks like the `adbusers` instructions are still included.

**Comment by Peter-Easton** [comment]
Whoa, whoops, sorry about that. Should be fixed now.

File: static/install.html [hunk 1]
```diff
@@ -82,6 +83,31 @@ <h2 id="prerequisites">
...
+            <p>Linux distributions that use udev for device management, such as Fedora, Ubuntu,
+            and Debian do not include and require correct udev rules in order to permit the
+            software on the computer to talk to the device if running adb or fastboot as a non-
+            root user. Running fastboot as non-root without the prerequisite udev rules installed
+            will result in fastboot hanging with a <code>waiting for device</code> error message
+            even with the device plugged in and seemingly properly configured.</p>
+
+            <p>The easiest way to obtain and add the correct udev rules to your computer is to
+            run the following commands to download the android rules, move them to the udev rules
+            directory and reload the rules, then add the <code>adbusers</code> usergroup and
```
Comment (thestinger) [inline static/install.html]: Need to remove this bit too.

**Review by thestinger — CHANGES_REQUESTED** [review]

File: static/install.html (hunk 1 above)
Comment (Peter-Easton) [reply static/install.html]: Whoops. I removed the part about the adbusers usergroup but left the intro paragraph and the part about Fedora and Debian.

**Comment by thestinger** [comment]
This is now covered for the supported operating systems.

**Pull request closed without merging**